sdk/storage/storage-file-share/test/fileclient.spec.ts

```diff
@@ -393,6 +437,38 @@ describe("FileClient", () => {
     assert.deepStrictEqual(result.contentDisposition, headers.fileContentDisposition);
   });
 
+  it("setHTTPHeaders with sddl permissions", async function () {
+    const filePermission =
+      "O:S-1-5-21-2127521184-1604012920-1887927527-21560751G:S-1-5-21-2127521184-1604012920-1887927527-513D:AI(A;;FA;;;SY)(A;;FA;;;BA)(A;;0x1200a9;;;S-1-5-21-397955417-626881126-188441444-3053964)";
+    await fileClient.create(content.length);
+    await fileClient.setHttpHeaders({},
+      {
+        filePermissionFormat: "Sddl",
```

Comment:
as there are 2 SDDL, might can make 1 has not format, so will use default SDDL

Reply:
Added one more case